### PR TITLE
Wallet tester

### PR DESCRIPTION
**Pull request by guibescos**
Many recipients of Pyth tokens have struggled to test their wallet setup since until now you could only "test" once you received the tokens but at that point it's too late and the tokens might be bricked.

This PRs goal is to introduce a new program called wallet tester that people can ping to test their Wallet Connect setup before receiving their tokens. 

Whenever someone is distributing locked tokens to someone, they can check if the wallet of the recepient has performed the test for extra safety.

### CODE REVIEW

File: frontend/pages/test.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            >
+              Click to test
+            </button>
+          </p>
+        ) : stakeConnection && hasTested ? (
+          <Description>
+            This wallet has already been tested succesfully.
+          </Description>
+        ) : (
+          <p className="p-2 hover:bg-hoverGray"> Please connect you wallet</p>
```
Comment (jayantk) [inline frontend/pages/test.tsx:64]: your

File: staking/programs/wallet-tester/output.rs [hunk 2]
```diff
@@ -0,0 +1,774 @@
+#![feature(prelude_import)]
```
Comment (jayantk) [inline staking/programs/wallet-tester/output.rs:1]: does this need to be checked in? it looks autogenned

**Review by jayantk — APPROVED** [review]
so this basically tests if they have the capability to interact with a solana program from their wallet? Seems like a good solution to me.

Have you deployed this to devnet? The vercel preview there doesn't seem to think the new program exists.

File: staking/programs/wallet-tester/output.rs (hunk 2 above)
Comment (guibescos) [reply staking/programs/wallet-tester/output.rs:1]: Thanks for catching this

File: frontend/pages/test.tsx (hunk 1 above)
Comment (guibescos) [reply frontend/pages/test.tsx:64]: Thanks

**Pull request merged**